File: data/conala-train_code-davinci-002_1666524963/Java/247.java
from bs4 import BeautifulSoup
import requests

url = 'https://www.google.com/search?q=java'

response = requests.get(url)
soup = BeautifulSoup(response.text, 'html.parser')

for meta in soup.find_all('meta'):
    if 'name' in meta.attrs and meta['name'] == 'description':
        print(meta['content'])

File: data/conala-train_code-davinci-002_1666524963/Python/247.py
import requests
from bs4 import BeautifulSoup

url = 'https://www.youtube.com/watch?v=9bZkp7q19f0'

r = requests.get(url)

soup = BeautifulSoup(r.text, 'html.parser')

title = soup.find('meta', attrs={'property': 'og:title'})
description = soup.find('meta', attrs={'property': 'og:description'})
image = soup.find('meta', attrs={'property': 'og:image'})

print(title['content'])
print(description['content'])
print(image['content'])